ansible/roles/ocp4-workload-iot-managed/readme.adoc

```diff
@@ -0,0 +1,128 @@
+= ocp4-workload-rhtr-iot-managed: Red Hat Tech Ready - Building an IoT application with Managed Services
```

Comment:
Please update the readme or any other references with the new role name

Reply:
Done https://github.com/redhat-cop/agnosticd/pull/2464/commits/ee88dccd765f154032d392902b0a37b03f8f4696#diff-e01f52f0c74b2cfd12e9b994c907fefc